Gemfile

```diff
@@ -1,2 +1,2 @@
 source 'https://rubygems.org'
-gem 'github-pages', group: :jekyll_plugins
\ No newline at end of file
+gem 'github-pages', group: :jekyll_plugins
```

Comment:
Nit: please keep the newline at the end of the file

Reply:
done! sorry for the extra work!